### PR TITLE
EditUserPopUp with Page Factory

### LINKED ISSUE #59: Rewriting Edit user pop up with Page Factory

### PR DESCRIPTION
**Pull request by Natalia62**

### CODE REVIEW

File: src/ui/components/edit_user_pop_up/edit_user_pop_up.py [hunk 1]
```diff
@@ -1,170 +1,110 @@
...
+            "new_password_input_element": ("xpath", "(//div[contains(@class,'item-control-input')]"
+                                                    "/span[contains(@class,'ant-input-password')"
+                                                    " and not (contains(@class,'login-box'))])[2]"),
+            "confirm_password_input_element": ("xpath", "(//div[contains(@class,'item-control-input')]"
+                                                        "/span[contains(@class,'ant-input-password')"
+                                                        " and not (contains(@class,'login-box'))])[3]"),
+            "submit_button": ("xpath", ".//button[contains(@class, 'submit-button')]")
+        }
+        self._wait = WebDriverWait(self.driver, 30)
+        self._actions = ActionChains(self.driver)
```
Comment (OksankaV) [inline src/ui/components/edit_user_pop_up/edit_user_pop_up.py]: We already have methods get_wait() and get_actions() for this in the CustomPageFactory class

File: src/ui/components/edit_user_pop_up/edit_user_pop_up.py [hunk 2]
```diff
@@ -1,170 +1,110 @@
...
 
     def click_on_user_type_button(self):
-        return self.user_type_button.click()
-
-    @property
-    def user_icon(self) -> WebElement:
-        if not self._user_icon:
-            self._user_icon = self.node.find_element(*USER_ICON)
-        return self._user_icon
+        return self.user_type_button.click
```
Comment (OksankaV) [inline src/ui/components/edit_user_pop_up/edit_user_pop_up.py]: Probably you missed () after click method in this row and other methods below this.
It is better to use PageFactory click_button() method, it waits element becomes clickable.

File: src/ui/components/edit_user_pop_up/edit_user_pop_up.py (hunk 1 above)
Comment (Natalia62) [reply src/ui/components/edit_user_pop_up/edit_user_pop_up.py]: Removed

File: src/ui/components/edit_user_pop_up/edit_user_pop_up.py (hunk 2 above)
Comment (Natalia62) [reply src/ui/components/edit_user_pop_up/edit_user_pop_up.py]: Rewrote

**Pull request merged**